hw/ip/aon_timer/doc/_index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "AON Timer Technical Specification"
+---
+
+# Overview
+
+This document specifies the Always-On ("AON") Timer IP functionality.
+This module conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification" >}})
+See that document for an overview of how it is integrated into the top level system.
+
+## Features
+
+- Two 32-bit upcounting timers: one timer functions as a wakeup timer, one as a watchdog timer
+- The watchdog timer has two thresholds: bark (generates an interrupt) and bite (resets core)
```

Comment:
do we need to make any clarifications about this not hooking up to NMI?
especially with the function of counting during low power?

Reply:
Not sure what you mean here, can you clarify?

Reply:
o just that the interrupt should not be hooked up to NMI at a system level.
Since that 'could' potentially cause ROM to get stuck due to this module being always on.

Reply:
Gotcha - I'll add a note